shell/platform/common/cpp/text_input_model.h

```diff
@@ -5,6 +5,8 @@
 #ifndef FLUTTER_SHELL_PLATFORM_CPP_TEXT_INPUT_MODEL_H_
 #define FLUTTER_SHELL_PLATFORM_CPP_TEXT_INPUT_MODEL_H_
 
+#include <codecvt>
+#include <locale>
```

Comment:
Why are these in the header? I don't see any uses outside of the implementation file.